tests/acceptance/features/coreApiWebdavProperties1/getQuota.feature

```diff
@@ -23,9 +23,9 @@ Feature: get quota
   @smokeTest
   Scenario Outline: retrieving folder quota when quota is set
     Given using <dav-path-version> DAV path
-    When the administrator sets the quota of user "Alice" to "10 MB" using the provisioning API
+    When the administrator sets the quota of user "Alice" to "10000" using the provisioning API
```

Comment:
let's use these steps:
- `Given user "Admin" has changed the quota of the "Alice Hansen" space to "10000"`
- `When user "Admin" changes the quota of the "Alice Hansen" space to "10000" owned by user "Alice"`

and remove provisioning ones (if can be). There will be confusion if we have multiple similar steps